tests/model/schema.js

```diff
@@ -1062,6 +1083,33 @@ describe( 'Schema', () => {
 				setData( model, '[]' );
 				expect( schema.checkAttributeInSelection( doc.selection, attribute ) ).to.be.false;
 			} );
+
+			it( 'should check attributes of the selection (selection at the beginning of the text)', () => {
+				setData( model, '<p><$text bold="true">[]foo</$text></p>' );
+				expect( schema.checkAttributeInSelection( doc.selection, 'italic' ) ).to.be.false;
```

Comment:
This is suspicious... why is it false? What gave the selection the bold attr? I guess `setData()` did, but I'd remove this test because it's confusing.